abci/types/types.proto

```diff
@@ -207,13 +207,13 @@ message ResponseCommit {
 // ConsensusParams contains all consensus-relevant parameters
 // that can be adjusted by the abci app
 message ConsensusParams {
-  BlockSizeParams block_size = 1;
+  BlockParams block = 1;
```

Comment:
Does this renaming justify a version bump of `ABCIVersion` in version.go?

Reply:
Good catch. I think this is technically a breaking change in ABCI